dev/com.ibm.ws.security.utility/src/com/ibm/ws/security/utility/tasks/BaseCommandTask.java

```diff
@@ -14,26 +14,70 @@
 
 import java.io.PrintStream;
 import java.text.MessageFormat;
+import java.util.ArrayList;
 import java.util.Enumeration;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 import java.util.TreeSet;
 
+import com.ibm.websphere.crypto.PasswordUtil;
+import com.ibm.websphere.crypto.UnsupportedCryptoAlgorithmException;
+import com.ibm.ws.crypto.util.AESKeyManager;
+import com.ibm.ws.crypto.util.SecureKeyFileParser;
+import com.ibm.ws.crypto.util.UnsupportedConfigurationException;
+import com.ibm.ws.kernel.productinfo.ProductInfo;
 import com.ibm.ws.security.utility.SecurityUtilityTask;
 import com.ibm.ws.security.utility.utils.CommandUtils;
 import com.ibm.ws.security.utility.utils.ConsoleWrapper;
+import com.ibm.wsspi.security.crypto.PasswordEncryptException;
 
 public abstract class BaseCommandTask implements SecurityUtilityTask {
 
     public static final String NL = System.getProperty("line.separator");
 
     protected final String scriptName;
 
+    static final String ARG_SECURE_KEY_FILE = "--secureKeyFile";
```

Comment:
```suggestion
    static final String ARG_SECURE_KEY_FILE = "--secureKeyFile";
```

change to aesConfigFile